azure/app-service/examples/advanced/.terraform.lock.hcl

```diff
@@ -0,0 +1,41 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
remove .terraform.lock.hcl

Reply:
Good spot, that's for a completely different module!